tools/install-powershell.ps1

```diff
@@ -51,8 +54,19 @@ if (-not $Destination) {
         $Destination = "${Destination}-daily"
     }
 }
-$Destination = $PSCmdlet.SessionState.Path.GetUnresolvedProviderPathFromPSPath($Destination)
-Write-Verbose "Destination: $Destination" -Verbose
+
+# Windows PowerShell doesn't have $PSCmdlet, so we'll just use the given path as-is
```

Comment:
Are you sure about that?

```powershell
# Win 5.1
function test {
>> [cmdletbinding()]
>> param()
>> $PSCmdlet | fl *
>> }
PS Z:\> test


ParameterSetName     : __AllParameterSets
MyInvocation         : System.Management.Automation.InvocationInfo
PagingParameters     :
InvokeCommand        : System.Management.Automation.CommandInvocationIntrinsics
Host                 : System.Management.Automation.Internal.Host.InternalHost
SessionState         : System.Management.Automation.SessionState
Events               : System.Management.Automation.PSLocalEventManager
JobRepository        : System.Management.Automation.JobRepository
JobManager           : System.Management.Automation.JobManager
InvokeProvider       : System.Management.Automation.ProviderIntrinsics
Stopping             : False
CommandRuntime       : test
CurrentPSTransaction :
CommandOrigin        : Runspace
```

Reply:
Hmm, this initially failed in my Windows PowerShell 5.1 console.  Let me see what's going on here.

Reply:
Just tried it and it seems fine, so don't know what happened previously.  Thanks for calling this out!